docker/sample_data/scylla/scylla_example.cql

```diff
@@ -5,16 +5,45 @@ CREATE TABLE if not exists users (
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (2, 'Jane', 38, 'jane@example.com', {'phone': '+1 (393) 214-4128', 'work_email': 'jane@example.com' }, true, null, {'FL', 'MA', 'CA'}, ['11:23:15']) IF NOT EXISTS;
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (3, 'Marguerite', 27, 'customer-2@example.com', { 'phone': '(472) 162-6435',  'work_email': 'customer-2-work@example.com' }, true, null, {'CA'}, ['08:12:54.123456789']) IF NOT EXISTS;
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (4, 'Lafayette', 55, 'customer-3@example.com', { 'phone': '+1 (074) 534-0949',  'work_email': 'customer-3-work@example.com' }, false, '2011-02-03', {'NY', 'TX'}, ['08:12:54.123']) IF NOT EXISTS;
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (5, 'Manuel', 23, 'customer-4@example.com', { 'phone': '+1 (831) 870-12348',  'work_email': 'customer-4@example.com' }, false, '2024-03-05', {'MO', 'KS'}, ['08:12:54.123456', '08:12:54.123456']) IF NOT EXISTS;
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (6, NULL, 23, 'customer-6@example.com', { 'phone': '+1 (831) 870-12349',  'work_email': 'customer-6@example.com' }, false, '2024-04-05', {'MO', 'NY'}, ['08:12:54.123456', '08:12:54.123456']) IF NOT EXISTS;
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (7, NULL, 26, 'customer-7@example.com', { 'phone': '+1 (831) 870-12359',  'work_email': 'customer-7@example.com' }, false, '2023-02-05', {'MO', 'TX'}, ['08:12:54.123456', '08:12:54.123456']) IF NOT EXISTS;
+INSERT INTO users (user_id, name, age, email, alternative_contacts, do_not_contact, last_contacted, states_lived, logins) VALUES (8, 'Jasmine', 34, 'customer-8@example.com', { 'phone': '+1 (831) 214-12359',  'work_email': 'customer-8@example.com' }, false, '2024-01-23', {'CA', 'NY'}, ['08:12:54.123456', '08:12:54.123456']) IF NOT EXISTS;
 
 
+CREATE TABLE if not exists user_activity (
+    user_id INT,
+    timestamp TIMESTAMP,
+    user_agent TEXT,
+    activity_type TEXT,
+    PRIMARY KEY (user_id, timestamp)
+);
```

Comment:
example of a table that relates to another table, as well as of a composite primary key